linera-core/benches/client_benchmarks.rs

```diff
@@ -71,7 +71,10 @@ where
         .unwrap()
         .unwrap();
 
-    chain2.receive_certificate(cert).await.unwrap();
+    chain2
+        .receive_certificate_and_update_validators(cert)
+        .await
+        .unwrap();
```

Comment:
Why are we changing the behavior of this test?

Reply:
I think we aren't: `receive_certificate` was already also updating the validators?